x/dispensation/utils/verfifysigner.go

```diff
@@ -5,21 +5,22 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/errors"
 
 	"github.com/Sifchain/sifnode/x/dispensation/types"
-	"github.com/cosmos/cosmos-sdk/x/bank"
-	"github.com/tendermint/tendermint/crypto"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
 )
 
-func VerifyInputList(inputList []bank.Input, pubKeys []crypto.PubKey) error {
+func VerifyInputList(inputList []banktypes.Input, pubKeys []*cdctypes.Any) error {
```

Comment:
I dont understand the reasoning behind using a pointer .
And also the reasoning behind using the Any type

Reply:
Not sure on the pointer but I believe Any is required because PubKey is an interface.